cmake_format/vscode_extension/src/extension.ts

```diff
@@ -55,43 +44,42 @@ export function activate(context: vscode.ExtensionContext) {
         console.log("Can't use cwd: " + cwd);
       }
 
-
       var env: any = {}
       if (config.get("mergeEnv", true)) {
         env = JSON.parse(JSON.stringify(process.env));
       }
+
       var cenv: any = config.get("env", {});
       if (cenv !== null) {
-        var delim = path.delimeter;
+        var delim = path.delimiter;
         for (var [key, value] of Object.entries(cenv)) {
           if (key.endsWith("PATH")) {
             var items = cenv[key].split(delim);
             if (key in env) {
-              items = items.concat(env[key].spit(delim));
+              items = items.concat(env[key].split(delim));
             }
             env[key] = items.join(delim);
           } else {
             env[key] = value;
           }
         }
       }
+
       if (!isEmpty(env)) {
         opts["env"] = env;
       }
 
-      const cp = require('child_process')
-
       // NOTE(josh): execFileSync will throw an Error if the
       // subprocess exits with non-zero status. The vscode GUI will
       // display the stacktrace in the corner.
-      var replacementText = cp.execFileSync(exePath, args, opts);
+      var replacementText = child_process.execFileSync(exePath, args, opts);
```

Comment:
This change looks arbitrary, what is the reason for it?

Reply:
it's inconsistent with how the other packages are defined.

Reply:
what other packages?

Reply:
`path` and `fs` [here](https://github.com/cheshirekow/cmake_format/blob/master/cmake_format/vscode_extension/src/extension.ts#L19-L20) and `vscode` [here](https://github.com/cheshirekow/cmake_format/blob/master/cmake_format/vscode_extension/src/extension.ts#L3). They're all using their full names.

Reply:
Oh I see what you mean.